Formula/serf.rb

```diff
@@ -4,8 +4,8 @@ class Serf < Formula
   desc "Service orchestration and management tool"
   homepage "https://serfdom.io/"
   url "https://github.com/hashicorp/serf.git",
-      :tag => "v0.7.0",
-      :revision => "0df3e3df1703f838243a7f3f12bf0b88566ade5a"
```

Comment:
I'd probably favour a blank line as suddenly yanking back the indent can look a bit weird, but there's no hard rule. If you felt more strongly than I do either way I'm happy to start enforcing it as such.